### PR TITLE
feat: APP-266 add sellOrderPrice to project document

### PR DESCRIPTION
**Pull request by blushi**
## Description

Exploring sanity option for https://regennetwork.atlassian.net/browse/APP-266
Check comment with explanation: https://regennetwork.atlassian.net/browse/APP-266?focusedCommentId=10687

---

### Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.*

I have...

- [x] provided a link to the relevant issue or specification
- [x] reviewed "Files changed" and left comments if necessary
- [x] added new items content to `./deskStructure.js`
- [ ] go through ["Deploying to production" instructions](https://github.com/regen-network/regen-sanity/blob/main/README.md#deploying-to-production) after this PR (and other PRs depending on this one, e.g. on regen-network/regen-web, if applicable) get merged

### Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**.*

I have...

- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed code correctness and readability
- [ ] reviewed documentation is accurate
- [ ] manually tested (if applicable)

### CODE REVIEW

File: schemas/documents/shared/project.js
Comment (blushi) [inline schemas/documents/shared/project.js:28]: @clevinson see my comment in https://regennetwork.atlassian.net/browse/APP-266?focusedCommentId=10687

File: schemas/objects/sellOrderPrice.js
Comment (blushi) [inline schemas/objects/sellOrderPrice.js:15]: this gives us more flexibility to put the price at the sell order level but if we think that we'll want to always define the same credit price for a given project, we could put this at the project document level cc/ @clevinson

File: schemas/documents/shared/project.js
Comment (clevinson) [reply schemas/documents/shared/project.js:28]: Sounds good. Thanks! I hadn't thought of that edge case.

File: schemas/objects/sellOrderPrice.js
Comment (clevinson) [reply schemas/objects/sellOrderPrice.js:15]: Makes sense to me.

**Review by r41ph — APPROVED** [review]

**Review by clevinson — APPROVED** [review]
pre-approving for once merge conflicts get resolved.

**Pull request merged**
